lib/countNumbersBetween.js

```diff
@@ -0,0 +1,11 @@
+const countNumbersBetween = (array, lower, upper) => {
+  count = 0
```

Comment:
Nice arrow function! Make sure to use `const` or `let` for all your variable declarations.